app/build.gradle

```diff
@@ -86,7 +87,7 @@ dependencies {
       'androidx.test.espresso:espresso-intents:3.1.0',
       'androidx.test.ext:junit:1.1.1',
       'androidx.test:runner:1.2.0',
-      'com.google.truth:truth:0.43',
+      'com.google.truth:truth:0.43'
```

Comment:
You can keep comma at the end of dependency. It will be easy for others to continue adding their dependencies.

Reply:
Resolved